hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -218,10 +225,17 @@ private boolean displayTable(ManagedRocksIterator iterator,
     }
 
     // Write to file output
-    try (PrintWriter out = new PrintWriter(new BufferedWriter(
-        new PrintWriter(fileName, UTF_8.name())))) {
-      return displayTable(iterator, dbColumnFamilyDef, out, schemaV3);
+    while (iterator.get().isValid()) {
+      try (PrintWriter out = new PrintWriter(new BufferedWriter(
+          new PrintWriter(fileName + fileSuffix, UTF_8.name())))) {
```

Comment:
If `preFileRecords` is not specified, we'd better make the filename the same as the previous filename (without `fileSuffix`)